app/helpers/task_list_helper.rb

```diff
@@ -58,7 +58,7 @@ def row_helper(trainee, task)
     when :school_details
```

Comment:
Should this be renamed to `:lead_partner_details`?

Reply:
It involves employing schools as well, which is why I was tempted to leave it alone. On the other hand, school_details isn't quite right either given that it could involve HEIs or SCITTs.